docs/en/ingest-management/elastic-agent/otel-agent-configuration.asciidoc

```diff
@@ -0,0 +1,52 @@
+[[otel-agent-configuration]]
+= {agent} OTel configuration
+
+preview::[]
+
+When running in `otel` mode, {agent} supports a standard OTel Collector configuration. 
+
+You can provide the OTel configuration to {agent} using any of the following four schemes:
+
+* `file` - The configuration is read from a file. For example, `file:path/to/config.yaml`.
+* `env` - The configuration is read from an environment variable. For example, `env:MY_CONFIG_IN_AN_ENVVAR`.
+* `yaml` - The configuration is read from a standard YAML-formatted file. For example, `yaml:exporters::debug::verbosity: detailed`.
+* `http` - The configuration is read from an HTTP URL. For example, `http://www.example.com`.
+// Perhaps we can improve these examples to make them more clear. I'm not sure I understand what the configuration would look like when passed as a URL, for example.
```

Comment:
my guess is a path to a yml file but i will check this

Reply:
Yes that's correct, e.g.:

```shell
./elastic-agent otel --config=http://localhost:8000/config.yaml
```